build.gradle

```diff
@@ -367,175 +365,74 @@ project(":ui") {
-                if(syst == "linux"){
-                    commandLine 'cp' , 'pipelib/libgenJNI.so', '.'
-                }
-            }
-            exec {
-                workingDir 'build/classes/test/pipelib'
-                commandLine 'cp' , '../../../../src/test/resources/edu/wpi/grip/ui/codegeneration/tools/pipe/AbsPipeline.h', 'pipe/'
-            }
-            exec {
-                workingDir 'build/classes/test'
-                commandLine 'cp', '../../../src/test/resources/edu/wpi/grip/ui/codegeneration/tools/realpipe/CMakeLists.txt', '.'
-            }
-            }
-        }
-    }
-
-    if(project.hasProperty('generation') || project.hasProperty('genonly')){
-        def syst = osdetector.os
-        test {
-            Platform platform = Installer.getPlatform()
-            Installer.setOpenCvVersion('3.1.0')
-            def jniLocation = project.getProperties().getOrDefault('jniLocation', platform.defaultJniLocation())
-            Installer.installJni("${jniLocation}")
-            def defaultLibPath = System.getProperty('java.library.path');
-            jvmArgs = ["-Djava.library.path=${defaultLibPath}${System.getProperty('path.separator')}${jniLocation}"]
-            if(project.hasProperty('genonly')){
-                useJUnit {
-                    includeCategories 'edu.wpi.grip.ui.codegeneration.GenerationTesting'
-                }
-            }
-        }
-    }
-
-    jfx {
-        mainClass = "edu.wpi.grip.ui.Main"
-        preLoader = "edu.wpi.grip.preloader.GripPreloader"
-
-        identifier = "GRIP"
-        appName = "GRIP"
-        vendor = "Worcester Polytechnic Institute"
-        nativeReleaseVersion = "${getVersionSimple()}-${arch}"
-
-        jfxMainAppJarName = "${jfx.appName}-${jfx.nativeReleaseVersion}.jar"
-
-        // -XX:-OmitStackTraceInFastThrow prevents the JIT from eating stack traces that get thrown a lot
-        // This is slower but means we actually get the stack traces instead of
-        // having them become one line like `java.lang.ArrayIndexOutOfBoundsException`
-        // and as such, would be useless.
-        // See: https://plumbr.eu/blog/java/on-a-quest-for-missing-stacktraces
-        // -Xmx limits the heap size. This prevents memory use from ballooning with a lot
-        // of JavaCV native objects being allocated hanging around waiting to get GC'd.
-        // -XX:MaxNewSize limits the size of the eden space to force minor GCs to run more often.
-        // This causes old mats (which take up little space on the heap but a lot of native memory) to get deallocated
-        // and free up native memory quicker, limiting the memory the app takes up.
-        jvmArgs = ["-XX:-OmitStackTraceInFastThrow", "-Xmx200m", "-XX:MaxNewSize=32m"]
-
-        bundleArguments = [
-                "linux.launcher.url": file('linuxLauncher/build/exe/linuxLauncher/linuxLauncher').toURI().toURL()
-        ]
-    }
-    mainClassName = jfx.mainClass
+//    task testSharedLib() {
```

Comment:
This task was never configured right. It actually does nothing and all of the `exec` statements would be run immediately (I think).

I left this here to help me figure out if I ported it wrong.